lib/functions-filters.php

```diff
@@ -10,4 +10,15 @@ function add_slug_body_class( $classes ) {
   }
   return $classes;
 }
-add_filter( 'body_class', 'add_slug_body_class' );
\ No newline at end of file
+add_filter( 'body_class', 'add_slug_body_class' );
+
+// Remove <p> tags around images and embeds in wp_content
```

Comment:
check out the styling. need to remove `<p>` so you can have images with margins outside the paragraphs like features in 8106